src/thunderbots/software/util/parameter/parameter.h

```diff
@@ -164,9 +132,7 @@ class Parameter
      *
      * @return An immutable reference to the Parameter registry
      */
-    static const std::map<std::string, std::pair<std::unique_ptr<std::mutex>,
-                                                 std::unique_ptr<Parameter<T>>>>&
-    getRegistry()
+    static const std::map<std::string, Parameter<T>*>& getRegistry()
```

Comment:
The string is just for the name right? We can get the name from the param as well, and since I just want to iterate over the list can this just return a vector of `Parameter<T>`?